loginas/utils.py

```diff
@@ -31,7 +33,17 @@ def login_as(user, request, store_original_user=True):
 
     # Log the user in.
     if hasattr(user, 'backend'):
+        signal_was_connected = False
+        if not la_settings.UPDATE_LAST_LOGIN:
+            # Prevent update of user last_login
+            signal_was_connected = user_logged_in.disconnect(update_last_login)
+
+        # Actually log user in
         login(request, user)
```

Comment:
Good catch, fixed in commit [60333cd](https://github.com/skorokithakis/django-loginas/pull/48/commits/60333cdf2e5a8533fdccd827dcbc429fac116bab) Thanks!